test/e2e/performanceprofile/functests/10_performance_ppc/ppc.go

```diff
@@ -60,10 +61,10 @@ func PPCTestCreateUtil() *PPCTestIntegration {
 	return p
 }
 
-var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform awareness", func() {
+var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform awareness", Label(string(label.PerformanceProfileCreator)), func() {
```

Comment:
the cast is awkward (why do we need it?), but we can clean up later

Reply:
Each set of labels is defined as a type (which is primitive is a string)
IIRC we agreed that the cast price for better separation between the different set of labels is worth it